src/flavors/pbdurham/config.json

```diff
@@ -1,4 +1,16 @@
 {
+  "flavor": {
```

Comment:
Assuming we'll have a `/flavor` endpoint at some point which returns this info.

Reply:
Sounds good. I'm working on a `/flavor` endpoint now 👍